src/api/store/settings.sagas.ts

```diff
@@ -27,6 +28,8 @@ export function* fetchSettingsWorker({ payload }: PayloadAction<string>) {
   const ciphertext: string = yield call(getFromStore, payload);
   if (ciphertext) {
     yield put(decryptSettings({ key: payload, value: ciphertext }));
+  } else {
+    yield put(rehydrateState({ key: payload, state: undefined }));
```

Comment:
I'm assuming this is for checking if the state was rehydrated? Maybe just add an action that adds the keys to `rehydratedKeys`, instead of resetting to initial state.

Reply:
This is already the case for this action. Do we want a separate one that doesn't actually rehydrate?